Schedule.API/Services/Procedures/EquipmentRelocationSchedulingService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using General.Repository;
+using Schedule.API.DTOs;
+using Schedule.API.Infrastructure.Repositories.Procedures.Interfaces;
+using Schedule.API.Model.Procedures;
+using Schedule.API.Services.Procedures.Interface;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Schedule.API.Services.Procedures
+{
+    public class EquipmentRelocationSchedulingService : IEquipmentRelocationSchedulingService
+    {
+        private readonly RepositoryWrapper<IExaminationRepository> _examinationWrapper;
+
+        public EquipmentRelocationSchedulingService(IExaminationRepository examinationRepository)
+        {
+            this._examinationWrapper = new RepositoryWrapper<IExaminationRepository>(examinationRepository);
+
+        }
+
+        public IEnumerable<int> CheckRoomsAvailability(EquipmentRelocationDto eqRealDto)
+        {
+            List<int> unavailableRoomsIds = new List<int>();
+            List<Examination> relocationRoomsExaminations = _examinationWrapper.Repository
```

Comment:
Maybe to use set for ids, because you can potentially add same id multiple times in list.

Reply:
You are right, I have changed it.